<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>me</class>
 <widget class="QMainWindow" name="me">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#me{
 background-color: rgb(255, 255, 255);
}
QFrame#frame{
border-bottom: 3px solid rgb(52, 103, 110);
}
QPushButton#pushButton{
background-color: rgb(52, 103, 110);
color: white;
font: bold 17px;
border-radius: 10px;
transition: 0.2s;
}
QPushButton#pushButton:hover {
background-color: rgb(72, 123, 130);
font: bold 19px;
}
QPushButton#pushButton:pressed {
background-color: rgb(62, 113, 120);
}
QPushButton#pushButtone:focus{
background-color: rgb(62, 113, 120);
box-shadow: 0 0 5px rgba(0,0,255,0.5);
font: bold 19px;
Width: 150px;
}
QPushButton#pushButton_2{
background-color: rgb(52, 103, 110);
color: white;
font: bold 17px;
border-radius: 10px;
transition: 0.2s;
}
QPushButton#pushButton_2:hover {
background-color: rgb(72, 123, 130);
font: bold 19px;
}
QPushButton#pushButton_2:pressed {
background-color: rgb(62, 113, 120);
}
QPushButton#pushButtone_2:focus{
background-color: rgb(62, 113, 120);
box-shadow: 0 0 5px rgba(0,0,255,0.5);
font: bold 19px;
Width: 150px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-80</x>
      <y>0</y>
      <width>1250</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(234, 234, 234);
background-color: rgb(132, 183, 190);
            height: 20px;
            width: 100%;
            top:0px;
            padding: 30px;
            position: fixed;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCommandLinkButton" name="commandLinkButton_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
border: 3px solid  rgb(52, 103, 110);
border-radius:9px;</string>
     </property>
     <property name="text">
      <string>Me</string>
     </property>
     <property name="icon">
      <iconset resource="navig.qrc">
       <normaloff>:/new/prefix1/709722.png</normaloff>:/new/prefix1/709722.png</iconset>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="description">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_3">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Jobs</string>
     </property>
     <property name="icon">
      <iconset resource="navig.qrc">
       <normaloff>:/icon-portfolio-20.png</normaloff>:/icon-portfolio-20.png</iconset>
     </property>
     <property name="shortcut">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_5">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>My Network</string>
     </property>
     <property name="icon">
      <iconset resource="navig.qrc">
       <normaloff>:/new/prefix1/pp.png</normaloff>:/new/prefix1/pp.png</iconset>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_4">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>10</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Massaging</string>
     </property>
     <property name="icon">
      <iconset resource="navig.qrc">
       <normaloff>:/885773_message_512x512.png</normaloff>:/885773_message_512x512.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="locale">
      <locale language="English" country="SouthSudan"/>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
     <property name="icon">
      <iconset resource="navig.qrc">
       <normaloff>:/new/prefix1/poster.png</normaloff>:/new/prefix1/poster.png</iconset>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="description">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Magneto</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(52, 103, 110);
color: white;
padding: 5px;
border-top-left-radius: 40px;
border-bottom-right-radius: 40px;</string>
     </property>
     <property name="text">
      <string>PROJOB</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>15</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(202, 243, 250);
color: rgb(0, 0, 0);
border-radius: 10px;
padding: 10px;
</string>
     </property>
     <property name="placeholderText">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_6">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>10</y>
       <width>135</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>My Network_2</string>
     </property>
     <property name="icon">
      <iconset resource="navig.qrc">
       <normaloff>:/new/prefix1/about-us-icon.png</normaloff>:/new/prefix1/about-us-icon.png</iconset>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton_7">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>10</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Jobs_2</string>
     </property>
     <property name="icon">
      <iconset resource="navig.qrc">
       <normaloff>:/icon-portfolio-20.png</normaloff>:/icon-portfolio-20.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>631</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 238);
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
background-color: rgb(255, 107, 255);
border-radius:13px</string>
     </property>
     <property name="text">
      <string>    Profile picture</string>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(61, 61, 61);
font: 12pt &quot;Rockwell&quot;;</string>
        </property>
        <property name="text">
         <string>Firstname:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_fname">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(61, 61, 61);
font: 12pt &quot;Rockwell&quot;;</string>
        </property>
        <property name="text">
         <string>Lastname:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_lname">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>631</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 238);
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(61, 61, 61);
font: 12pt &quot;Rockwell&quot;;</string>
        </property>
        <property name="text">
         <string>Employment:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_lname_3">
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>69</y>
       <width>601</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Rockwell</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>49</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(61, 61, 61);
font: 12pt &quot;Rockwell&quot;;</string>
     </property>
     <property name="text">
      <string>Biography:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>450</y>
      <width>161</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: white;
padding: 6px;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Add Company</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>450</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: white;
padding: 6px;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Change Information</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1142</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="navig.qrc"/>
 </resources>
 <connections/>
</ui>
